### PR TITLE
log: default to rails

### PR DESCRIPTION
**Pull request by Sirupsen**
@fbogsany instead of requiring overriding this for Rails, let's default to a sane default. We already have a bunch of `defined?(::Rails)` in here and this helps me tremendously to simplify our configuration.

### CODE REVIEW

**Comment by arthurnn** [comment]
👍

**Comment by fw42** [comment]
lol @arthurnn

File: test/test_helper.rb [hunk 1]
```diff
@@ -5,3 +5,17 @@
 require 'active_support/test_case'
 
 require 'memcached_store'
+
+class Rails
+  def self.logger
+    @logger ||= Logger.new("/dev/null")
+  end
+
+  def self.env
```
Comment (fw42) [inline test/test_helper.rb]: why is this needed?

Comment (Sirupsen) [inline test/test_helper.rb]: https://github.com/Shopify/memcached_store/blob/master/lib/active_support/cache/memcached_store.rb#L25

**Comment by fbogsany** [comment]
LGTA :ship:

**Comment by Sirupsen** [comment]
LGTALGTM

**Comment by fw42** [comment]
LSTFUTM

**Comment by Sirupsen** [comment]
Looks Sound To Flo Und die Tiermesse?

**Comment by arthurnn** [comment]
lol
